cmd/tapcli/universe.go

```diff
@@ -698,6 +715,218 @@ func universeFederationDel(ctx *cli.Context) error {
 	return nil
 }
 
+var universeFederationConfigCommand = cli.Command{
+	Name:      "config",
+	ShortName: "c",
+	Usage:     "Change the sync behavior of the local Universe",
+	Description: `
+	Manage the sync behavior of the local Universe. These settings are
+	defined by the proof type (issuance or transfer), the sync behavior
+	(insert from remote Universe or export to remote Universe), and the
+	scope (all assets or specific assets).
+        `,
+	Subcommands: []cli.Command{
+		universeFederationGlobalConfig,
+		universeFederationLocalConfig,
+	},
+}
+
+var (
+	proofInsertName        = "insert"
+	proofExportName        = "export"
+	universeSyncConfigArgs = []cli.Flag{
+		cli.StringFlag{
+			Name:  proofTypeName,
+			Usage: "the type of proof",
+		},
+		cli.StringFlag{
+			Name: proofInsertName,
+			Usage: "if true, remote Universes can push proofs to" +
+				"the local Universe",
+		},
+		cli.StringFlag{
+			Name: proofExportName,
+			Usage: "if true, remote Universes can pull proofs from" +
+				"the local Universe",
+		},
+	}
+)
+
+func parseConfigArgs(ctx *cli.Context,
+	isGlobal bool) (*unirpc.GlobalFederationSyncConfig,
+	*unirpc.AssetFederationSyncConfig, error) {
+
+	var (
+		insertOpt *bool
+		exportOpt *bool
+		err       error
+	)
+
+	// The cli.BoolFlag works for variables that are false by default,
```

Comment:
I think we can use `BoolTFlag` here? If we want it to be true by default.

Reply:
I think we don't want it to have a default at all and have it be fully explicit.

Something like `tapcli universe federation config global --proof_type=transfer --insert true --export false`

Where `insert` and `export` without a bool fail.

Reply:
Ah sure I think that makes sense.